src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/Args.java

```diff
@@ -126,13 +125,27 @@
    * @return [key1=value1,key2=value2,...], [] if keyValueMapping=empty/null
    */
   public static List<String> keyValues(Map<?, ?> keyValueMapping) {
+    return namedKeyValues("", keyValueMapping);
+  }
+
+  /**
+   * Produces a named key/value pair list given a name and a {@link Map}.
+   *
+   * @return [--name, key1=value1, --name, key2=value2,...], [] if keyValueMapping=empty/null
+   */
+  public static List<String> namedKeyValues(String name, Map<?, ?> keyValueMapping) {
     List<String> result = Lists.newArrayList();
```

Comment:
maybe this comment is obsolete given your other one, but if we invert this then the zipping would require an extra iteration over the list that has already been formed. This way, its done 'in place' during the iteration.